source/simulator/parameters.cc

```diff
@@ -318,6 +318,32 @@ namespace aspect
                        "the composition system gets solved. See 'linear solver "
                        "tolerance' for more details.");
 
+    prm.enter_subsection ("Solver parameters");
+    {
+      prm.enter_subsection ("Newton solver parameters");
+      {
+        prm.declare_entry ("Nonlinear Newton solver switch tolerance", "1e-5",
+                           Patterns::Double(0,1),
+                           "A relative tolerance up to which the nonlinear Piccard solver "
+                           "will iterate, before changing to the newton solver. This parameter "
+                           "is only relevant if Nonlinear solver scheme is set to a Newton type "
```

Comment:
I guess that I can now remove that sentence at all, now it is put in a separate subsection.